arrow/parser.py

```diff
@@ -175,7 +175,7 @@ def _parse_token(self, token, value, parts):
         elif token in ['DD', 'D']:
             parts['day'] = int(value)
 
-        elif token in ['HH', 'H']:
+        elif token in ['HH', 'H', 'hh', 'h']:
```

Comment:
On Wed, Jan 07, 2015 at 05:37:44PM -0800, Bertrand Janin wrote:
>    In arrow/parser.py:
> 
>  > @@ -175,7 +175,7 @@ def _parse_token(self, token, value, parts):
>  >          elif token in ['DD', 'D']:
>  >              parts['day'] = int(value)
>  > 
>  > -        elif token in ['HH', 'H']:
>  > +        elif token in ['HH', 'H', 'hh', 'h']:
> 
>    But that would break 's' and 'S'.

Depends if you do at the top an ``token = token.upper()`` or just adjust the
if/elif statements to be ``elif token.upper() in ['HH', 'H']``